### DISCUSSION TITLE
我觉得这个项目别再更新比较好

### DISCUSSION THREAD

**Discussion by jingzz1** [WeNeedHome/SummaryOfLoanSuspension#1023] [General]
一方面，国内断供声明已经无法通过正规渠道发出来了，热搜也被压下去，另一方面，面对300+的断供声明政府和银行都不痛不痒，没做任何表示，那么再增加断供声明也没用。
如果因为买到烂尾楼而一无所有，那就断供吧，断供的最坏结果也只是上征信而已。

**Comment by fiytodie** [comment]
？？？断供了首付和之前付的房贷也没了，上征信对很多人只是微不足道的损失，如果说一直烂尾断也就断了，要是后面继续弄好了，结果房产被银行收走了，那不得气死
不管怎么说断供终归只是给上面一点压力，希望能明确一个解决方案吧，别搞什么威不可测

**Reply by jingzz1** [reply]
参考过去10多年，卖得差不多的期房，烂尾了基本没有盘活的可能，除非房价再涨一倍，涨接手的开发商有利可图，这样才有可能盘活。

**Reply by fiytodie** [reply]
是的，我老家有个10年多的烂尾楼，舅伯买了，开发商换了3,4个，还在烂着，后面还出现过因为当初的价格覆盖不了现在的建筑费要加2000/平的，结果收不上来就彻底摆烂了
但是现在大部分所谓的烂尾，都没烂的彻底，只是延迟交房，就是总数太多了，暴雷的房企也多，后续肯定会有少部分是真正烂尾的，所以都希望优先处理自己的，就像那个“高学历”楼盘一样，就看谁更能闹，更新项目终归也是闹的一种方式，上了名单总归会优先处理吧

**Comment by dalezhang93** [comment]
不痛不痒是不可能的, 有银行通知抓着装了,很是诧异,大热天不让穿短裤.我的贷款银行提前还款困难重重,顶着黄码折腾了好几天把事办了

**Comment by clleo888** [comment]
面对300+的断供声明政府和银行都不痛不痒，没做任何表示，
你的结论是 : 那么再增加断供声明也没用。
为何结论不能是 : 再继续增加断供声明直至他们痛点?
但我認為如果沒那麼多烂尾, 你想增加也增加不了, 為何就不能讓這裡真實反映有多少烂尾樓,有多少報告多少,不要人為控制了...

**Reply by jingzz1** [reply]
> 面对300+的断供声明政府和银行都不痛不痒，没做任何表示， 你的结论是 : 那么再增加断供声明也没用。 为何结论不能是 : 再继续增加断供声明直至他们痛点?

你有没有想过，300+后没怎么增加，不是因为后来没有楼盘断供，而是断供声明发不出来。

**Reply by clleo888** [reply]
发不出来和不去发是两回事, 这贴的意思是不去发, 我没有理解错吧!?
更何况想发出来的方法有很多, 你又怎样确定是一定发不出来呢?

**Comment by fangqiangqiang007** [comment]
主要是现在三百多个楼盘好像轰轰烈烈，但是有几个人真的断供了，一句上征信不够，再来一句影响你孩子，吓得99.99%的人恨不得连夜借钱还贷，谁会把你这个断供声明当回事

**Reply by zxcs001** [reply]
所以说大部分国人活该呗，人不自救，天也难佑。

我这边给我烂尾了一套小别墅，19年末就该交房一直烂到现在。

知道我怎么做的？先把贷款能借银行的钱，能刷的信用卡全刷爆。然后自己的房子一卖，全部搞到国外去。

征信？你征信能管得着我？开玩笑？下一次回国，我是外国护照签证入关，你这个户口本上的那个人，我不认识，你爱找谁找谁去。

孩子上学？抱歉啊，我第一没孩子，第二即使有孩子，你征信也拿捏不到我啊，我在你手里没有人质啊。

**Comment by TALK1949** [comment]
不痛不痒却开始出现拟定针对烂尾楼的救市政策？
不痛不痒是不可能的，官方想让你觉得不痛不痒，放弃继续扩大停贷规模才是目的。
我们要做的只是多传播，仅此而已，至于结果，那要看业主们决心了。

**Reply by jingzz1** [reply]
> 不痛不痒却开始出现拟定针对烂尾楼的救市政策？ 不痛不痒是不可能的，官方想让你觉得不痛不痒，放弃继续扩大停贷规模才是目的。 我们要做的只是多传播，仅此而已，至于结果，那要看业主们决心了。

官方那所谓的救市就是提供3000亿房地产基金……3000亿进入楼市，肯定会进一步抬高房价，烂尾楼的补救成本最终还是分摊到民从身上！
